src/main/java/org/spongepowered/api/Server.java

```diff
@@ -206,6 +206,15 @@
     Optional<WorldProperties> createWorld(WorldCreationSettings settings);
 
     /**
+     * Creates a copy of the provided world under the new name given and returns
+     * the new copy if the copy was possible
+     *
+     * @param originalWorld The original world instance to copy
+     * @param copyName The name that should be used in the duplicated entry instead of the originalWorld's name.
+     */
+    Optional<World> copyWorld(World originalWorld, String copyName);
```

Comment:
Might be good to change this to `WorldProperties` rather than `World` so that you don't have to load a world to copy it.